src/index.css

```diff
@@ -15,3 +15,8 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/* width */
+::-webkit-scrollbar {
+  width: 0px;
+}
```

Comment:
should this be here? and what is purpose of this

Reply:
I tought it would be better to write all css (in case Tailwind can't solve the problem) in one document. But its not problem to move this code into special CSS file for sidebar.